enterprise/frontend/src/embedding-sdk/components/public/StaticDashboard/StaticDashboard.tsx

```diff
@@ -28,18 +28,23 @@ export type StaticDashboardProps = {
 
   navigateToNewCardFromDashboard?: (
     opts: NavigateToNewCardFromDashboardOpts,
-  ) => void; // TODO: should not be part of publicly exposed api
+  ) => void;
 };
 
-const _StaticDashboard = ({
+export type StaticDashboardProps = Omit<
+  StaticDashboardInnerProps,
+  "navigateToNewCardFromDashboard"
+>;
+
+export const _StaticDashboard = ({
```

Comment:
This could work, but if the name of the props is `StaticDashboardInnerProps` wouldn't it be better to name the component `StaticDashboardInner` rather than `_StaticDashboard`? Just to be consistent with how we name props in other places. What do you think?